src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -359,6 +359,109 @@ public static String csvToHtmlTable(String str) {
                              result.toString());
     }
 
+    private static List<String> combineRowDataEntries(List<String> rowData,
+            String delimiter) {
+        String combinedEntries = rowData.get(0) + delimiter + rowData.get(1);
+        rowData = new ArrayList<String>();
+        rowData.add(combinedEntries);
+        return rowData;
+    }
+    
+    private static String wrapAsRegExp(String str) {
+        String wrapper = "^%s: .*";
+        return String.format(wrapper, str);
+    }
+
+    /**
+     * Convert a csv string to a beautified html table string for displaying
+     * 
+     * @param str
+     * @return beautified html table string
+     */
+    public static String csvToBeautifiedHtmlTable(String str) {
```

Comment:
This function is quite long and hard to understand. Could you find a more elegant way of writing this?